OpenSim/Simulation/Model/ExpressionBasedBushingForce.cpp

```diff
@@ -609,7 +605,7 @@ void ExpressionBasedBushingForce::updateDisplayer(const SimTK::State& s)
                                                  globalLocation1);
     _model->getSimbodyEngine().transformPosition(s, body2, get_location_body_2(),
                                                  globalLocation2);
-
+    /*
```

Comment:
@sherm1 You're right I'll remove all these. I kept some around in development to reuse/replicate in generateDecorations when it wasn't implemented but it's not worth it with the added clutter.